src/client/common/installer/productInstaller.ts

```diff
@@ -126,6 +126,54 @@ export abstract class BaseInstaller {
             .install(product, resource, cancel, reInstallAndUpdate);
     }
 
+    public async getVersion(product: Product, resource?: InterpreterUri): Promise<string | undefined> {
```

Comment:
Did using the pythonAPI function not work here?

Reply:
This is the reason:
![image](https://user-images.githubusercontent.com/20347396/123492534-9a155280-d5ce-11eb-9c24-ad20b018d102.png)

I think the python api uses the bottom command, and it gives the version of something different (its ipython) and that wouldn't work. I'll check their code, but for this PR we might want to do it ourselves. 

And fix it for Conda of course.

Reply:
For modules (which ipykernel should be treated as), I believe the Python API uses 
```
python -c 'import ipykernel; print(ipykernel.__version__)'
```

which would yield the right result

![image](https://user-images.githubusercontent.com/30305945/123531889-21d98a80-d6bd-11eb-84b0-b99fa7840017.png)

https://github.com/microsoft/vscode-python/blob/362eb34cafcbc513919eb4d78ea0521a0ed6738a/src/client/common/process/internal/python.ts#L104-L112